api/internal/managers/app/config/manager.go

```diff
@@ -14,8 +14,9 @@ var _ AppConfigManager = &appConfigManager{}
 
 // AppConfigManager provides methods for managing appConfigstructure setup
 type AppConfigManager interface {
```

Comment:
Happy to keep it separate since you're no longer touching that code but we can remove this mutex since it's unused I think.
https://github.com/replicatedhq/embedded-cluster/blob/fefb871ad0751c8dc37a75b81043b2c514c37093/api/internal/managers/app/config/manager.go#L26